lua/nvim-treesitter/utils.lua

```diff
@@ -17,6 +17,15 @@ function M.setup_commands(mod, commands)
   end
 end
 
+function M.get_path_sep()
+  local path_sep = '/'
+  if fn.has('win32') == 1 then
+    path_sep = '\\'
+  end
+
+  return path_sep
+end
+
```

Comment:
We should have somtehing like `push_to_path` looking like `M.push_to_path(path, ...)` to avoid having the `path .. path_sep .. 'foo'` pattern everywhere.

Reply:
maybe just join ?